tests/performance_tests/performance_test_basic.py

```diff
@@ -236,16 +232,34 @@ def fileOpenMode(self, filePath) -> str:
             append_write = 'w'
         return append_write
 
+    def isImportantTransaction(self, transaction):
+        if self.clusterConfig.nodeosVers == "v2":
+            return True
+        else:
+            if transaction['actions'][0]['account'] != 'eosio' or transaction['actions'][0]['action'] != 'onblock':
+                return True
+        return False
+
     def queryBlockTrxData(self, node, blockDataPath, blockTrxDataPath, startBlockNum, endBlockNum):
-        for blockNum in range(startBlockNum, endBlockNum):
+        for blockNum in range(startBlockNum, endBlockNum + 1):
+            blockCpuTotal, blockNetTotal, blockTransactionTotal = 0, 0, 0
             block = node.fetchBlock(blockNum)
             btdf_append_write = self.fileOpenMode(blockTrxDataPath)
             with open(blockTrxDataPath, btdf_append_write) as trxDataFile:
-                self.clusterConfig.writeTrx(trxDataFile, block, blockNum)
-
+                for transaction in block['payload']['transactions']:
+                    if self.isImportantTransaction(transaction):
```

Comment:
This should be called `isOnBlockTransaction` and be changed to `if not self.isOnBlockTransaction(transaction)`

Reply:
Addressed: [3127329](https://github.com/AntelopeIO/leap/pull/945/commits/31273296fee664f68d6978efa136db27a0df5ac4)

Reply:
And here: [cd87f42](https://github.com/AntelopeIO/leap/pull/945/commits/cd87f42a9b5f672a98afc978403c960ddad6f332)